drivers/gles2/shaders/tonemap.glsl

```diff
@@ -138,7 +142,7 @@ uniform ivec2 glow_texture_size;
 vec4 texture2D_bicubic(sampler2D tex, vec2 uv, int p_lod) {
 	float lod = float(p_lod);
 	vec2 tex_size = vec2(glow_texture_size >> p_lod);
-	vec2 pixel_size = vec2(1.0) / tex_size;
+	vec2 pix_size = vec2(1.0) / tex_size;
 
```

Comment:
Could we maybe use something like e.g. `pixel_size` and `inv_pixel_size`, or `texel_size` if this is specific to this routine?

Having `pixel_size` and `pix_size` doesn't make it clear what either is referring to without looking them up.

Reply:
I agree. texel_size would be a more appropriate name here. And then it is distinguished from pixel_size.